x-pack/plugins/apm/public/components/shared/charts/spark_plot/index.tsx

```diff
@@ -50,7 +50,7 @@ export function SparkPlot({
   valueLabel: React.ReactNode;
   compact?: boolean;
   comparisonSeries?: Coordinate[];
-  comparisonSeriesColor: string;
+  comparisonSeriesColor?: string;
```

Comment:
I'm always a little sceptical when a required argument in a shared component is made optional. This weakens the typing, so there should be a good reason for doing this

Reply:
You can also use a union type with null. It's a little more verbose but at least it's explicit and it's a lot harder to forget about it.